tools/cloudera/build_hue_common.sh

```diff
@@ -108,7 +108,7 @@ function redhat8_ppc_install() {
     # NODEJS 14 install
     sudo -- sh -c 'yum install -y nodejs npm'
     # Pip modules install
-    sudo pip38_bin=${pip38_bin} -- sh -c '${pip38_bin} install virtualenv virtualenv-make-relocatable mysqlclient==2.1.1'
+    sudo pip38_bin=${pip38_bin} -- sh -c '${pip38_bin} install virtualenv==20.24.4 virtualenv-make-relocatable==0.0.1 mysqlclient==2.1.1'
```

Comment:
Can we define `$VIRTUALENV_VERSION` a the top of this file or in build_hue_common.sh or the Makefile itself? That should make it easier to change in the future.

Reply:
Created variables($VIRTUAL_ENV_VERSION, $VIRTUAL_ENV_RELOCATABLE_VERSION) to hold the versions